### PR TITLE
Enforce case requires else; allow else nil

### PR DESCRIPTION
**Pull request by bf4**

### CODE REVIEW

File: lib/active_model/serializer/include_tree.rb [hunk 1]
```diff
@@ -68,6 +68,8 @@ def [](key)
           self.class.new(@hash[:*])
         when @hash.key?(:**)
           self.class.new(:** => {})
+        else
+          nil
```
Comment (bf4) [inline lib/active_model/serializer/include_tree.rb]: Missing this else would now fail Rubocop

Comment (beauby) [inline lib/active_model/serializer/include_tree.rb]: Great!

**Comment by beauby** [comment]
LGTM, will merge once the tests are green.

**Comment by bf4** [comment]
green

**Pull request merged**